client/src/pages/Booking/Booking.tsx

```diff
@@ -0,0 +1,116 @@
+import { useState } from 'react';
+import { useHistory } from 'react-router-dom';
+
+import { DatePicker, Day, MuiPickersUtilsProvider } from '@material-ui/pickers';
+import { MaterialUiPickersDate } from '@material-ui/pickers/typings/date';
+import { MuiThemeProvider } from '@material-ui/core';
+import { createMuiTheme } from '@material-ui/core';
+import DateFnsUtils from '@date-io/date-fns';
+import CircularProgress from '@material-ui/core/CircularProgress';
+import Grid from '@material-ui/core/Grid';
+import Card from '@material-ui/core/Card';
+
+import { useAuth } from '../../context/useAuthContext';
+import NavBar from '../../components/NavBar/NavBar';
+import BookingItem from '../../components/BookingItem/BookingItem';
+import useStyles from './useStyles';
+
+const materialTheme = createMuiTheme({
+  overrides: {
+    MuiPickersCalendarHeader: {
+      switchHeader: {
+        color: '#ff4545',
+      },
+      daysHeader: {
+        justifyContent: 'space-around',
+      },
+    },
+    MuiPickersDay: {
+      daySelected: {
+        color: '#ffffff',
+        backgroundColor: '#ff4545',
+      },
+    },
+    MuiPickersBasePicker: {
+      pickerView: {
+        maxWidth: '100%',
+      },
+    },
+    MuiPickersCalendar: {
+      week: {
+        justifyContent: 'space-around',
+      },
+    },
+  },
+});
+
+function Booking(): JSX.Element {
+  const classes = useStyles();
+
+  const { loggedInUser } = useAuth();
+  const history = useHistory();
+
+  const [bookedDays] = useState<number[]>([21, 25]);
+  const [selectedDate, setDate] = useState<MaterialUiPickersDate>(new Date());
+
+  if (loggedInUser === undefined) return <CircularProgress />;
+  if (!loggedInUser) {
```

Comment:
Good thinking! We won't need this check though as we will eventually have a protected route which will take care of this for us :)